actionpack/test/controller/rate_limiting_test.rb

```diff
@@ -47,7 +47,7 @@ class RateLimitingTest < ActionController::TestCase
     get :limited_to_two
     assert_response :ok
 
-    sleep 2.1
+    sleep 3
```

Comment:
Maybe could use [AS travel](https://edgeapi.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel) here to avoid adding 3 seconds to this one test case?

Reply:
Can't do that because the expiry is set in Redis, not in Rails.